backend/Piece.py

```diff
@@ -1,16 +1,16 @@
 from enum import Enum
 
+from backend.Cell import Color
+
 
 class Turn(Enum):
-    BLACK = 1
-    WHITE = 2
 
     @classmethod
     def swap_turn(self, curr):
```

Comment:
Honestly, I think swap_turn could be migrated to the Color enum too, if we just want to have everything located in one spot.

If we do migrate it, it might be worth swapping the `else` to an `elif curr == Color.WHITE`. The current turn shouldn't ever be listed as VOID or EMPTY, but it might be worth checking and throwing an error for anyway.

Reply:
updated.